packages/wdio-browserstack-service/src/insights-handler.ts

```diff
@@ -864,12 +864,15 @@ class _InsightsHandler {
     }
 
     private getIntegrationsObject () {
+        const caps = (this._browser as WebdriverIO.Browser)?.capabilities as WebdriverIO.Capabilities
+        const sessionId = (this._browser as WebdriverIO.Browser)?.sessionId
+
         return {
-            capabilities: this._platformMeta?.caps,
-            session_id: this._platformMeta?.sessionId,
-            browser: this._platformMeta?.browserName,
-            browser_version: this._platformMeta?.browserVersion,
-            platform: this._platformMeta?.platformName,
+            capabilities: caps,
+            session_id: sessionId,
```

Comment:
Please ensure to raise this on v7 too and check it works for w3c and non w3c mode.

Reply:
Have created [V8](https://github.com/webdriverio/webdriverio/pull/12368) and [V7](https://github.com/webdriverio/webdriverio/pull/12369) PRs

Reply:
The code that is changed is already present, so it's already working for W3c and non W3c